src/client/game.cpp

```diff
@@ -2943,6 +2943,8 @@ void Game::handleClientEvent_SetSky(ClientEvent *event, CameraOrientation *cam)
 	if (event->set_sky->type == "regular") {
 		// Shows the mesh skybox
 		sky->setVisible(true);
+		// Orbit Tilt:
+		sky->setBodyOrbitTilt(event->set_sky->body_orbit_tilt);
```

Comment:
Could this call be moved out of the if() to avoid code duplication?

Reply:
Yes, I think it can. I'll go and do that.